package.json

```diff
@@ -37,10 +42,10 @@
     "source-map-support": "^0.4.0"
   },
   "devDependencies": {
-    "autoprefixer": "^6.1.2",
     "babel-core": "^6.3.17",
```

Comment:
We seem to inconsistently apply versioning in this file. What do we think should we be less restrictive and keep using `^xxx.xxx.xxx` or lock things down to a specific version? If we decided to go down the  `^xxx.xxx.xxx` we could also use something like shrinkwrap.

I think we should be more restrictive, the decision to move to a new version no matter how small should be a deliberate decision. Mainly because I don't trust people and their interpretation/implementation of semver.